modules/ROOT/partials/configuration/autocorrect_service_url.adoc

```diff
@@ -0,0 +1,21 @@
+[[autocorrect_service_url]]
+
+== `+autocorrect_service_url+`
+
+This option specifies the URL of the server-side `+ephox-spelling+` service. For instructions on how to set up a Spellchecking server-side component, see: the xref:introduction-to-premium-selfhosted-services.adoc[server-side component installation guide].
+
+NOTE: If not specified, the plugin will attempt to fallback onto `autocorrect_rpc_url` option.
+
+*Type:* `+String+`
+
+*Default value:* `+''+`
+
+=== Example: Using `+autocorrect_service_url+`
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  autocorrect_service_url: 'http://mydomain.com/spelling/'
```

Comment:
```suggestion
  autocorrect_service_url: 'http://mydomain.com/ephox-spelling/'
```
This is the name of the WAR file distributed, so we should prefer using it as it'll likely be what's needed by the customer.